### PR TITLE
Add Binance presence

### PR DESCRIPTION
**Pull request by Hibanka**
Website: [Binance](https://www.binance.com/)

### CODE REVIEW

File: websites/B/Binance/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "research.binance.com"
+  ],
+  "description": {
+    "en": "The world's largest crypto exchange",
+    "ru": "Крупнейшая в мире криптобиржа"
+  },
+  "service": "Binance",
+  "version": "1.0.0",
+  "logo": "https://i.postimg.cc/FKVWP1tZ/logo.png",
+  "thumbnail": "https://i.postimg.cc/Xqyg6PCL/ogImage.jpg",
```
Comment (Bas950) [inline websites/B/Binance/dist/metadata.json:21]: upload to imgur

File: websites/B/Binance/presence.ts [hunk 2]
```diff
@@ -0,0 +1,93 @@
+const presence = new Presence({
+  clientId: "807591728759570453"
+})
+
+const browsingStamp = Math.floor(Date.now() / 1000)
```
Comment (Bas950) [inline websites/B/Binance/presence.ts:5]: merge consts

File: websites/B/Binance/presence.ts [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+    presenceData.startTimestamp = browsingStamp;
+    presenceData.details = "Binance Launchpad";
+  } else if (window.location.host.startsWith("research")) {
+    presenceData.startTimestamp = browsingStamp;
+    presenceData.details = "Binance Research";
+  }
+
+  if (presenceData.details == null) {
+    presenceData.startTimestamp = browsingStamp;
+    presenceData.details = "Browsing...";
```
Comment (Bas950) [inline websites/B/Binance/presence.ts:89]: you are not setting the data afterwords...

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by Bas950 — CHANGES_REQUESTED** [review]
Fix codacy issues

**Review by ririxidev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]
Fix codacy issues

**Pull request merged**